modules/auth/handle_genesis.go

```diff
@@ -5,13 +5,10 @@ import (
 	"fmt"
 
 	tmtypes "github.com/tendermint/tendermint/types"
-
-	"github.com/rs/zerolog/log"
 )
 
 // HandleGenesis implements modules.GenesisModule
 func (m *Module) HandleGenesis(_ *tmtypes.GenesisDoc, appState map[string]json.RawMessage) error {
-	log.Debug().Str("module", "auth").Msg("parsing genesis")
```

Comment:
Can we bring back those logs please? I think they are interested also when parsing the genesis by hand